aspnetcore/fundamentals/use-http-context/samples/Program.cs

```diff
@@ -0,0 +1,167 @@
+#define RequestHeaders // Default CREATE P1 PM PE I1 I0 IP CERT CERT2 CERT3 RE CONFIG LOG #i REB 
+// CONFIGB LOGB IWHB DEP R1 LE LF IM SM NR NR2 RP WILD PBG PBP EPB OP1 OP2 OP3 OP4
+// CB BA CJSON MULTI STREAM XTN AUTH1 AUTH2 AUTH3 AUTH4 CORS CORS2 SWAG SWAG2 
+// FIL2 IHB CHNGR ADDMID
+#if NEVER
+#elif RequestHeaders
+#region snippet_RequestHeaders
```

Comment:
After this PR mergers I'll probably replace `#region` with
```
 // <snippet_1>
code
// </snippet_1>
```
 syntax. @blowdart gets mad when he see's regions.